kivymd/uix/behaviors/touch_behavior.py

```diff
@@ -1,51 +1,34 @@
 """
 Behaviors/Touch
 ===============
-
 .. rubric:: Provides easy access to events.
-
 The following events are available:
-
 - on_long_touch
 - on_double_tap
 - on_triple_tap
-
 Usage
 -----
-
 .. code-block:: python
-
     from kivy.lang import Builder
-
     from kivymd.app import MDApp
     from kivymd.uix.behaviors import TouchBehavior
     from kivymd.uix.button import MDRaisedButton
-
     KV = '''
     Screen:
-
         MyButton:
             text: "PRESS ME"
             pos_hint: {"center_x": .5, "center_y": .5}
     '''
-
-
```

Comment:
The double indentation before the class declaration is the PEP8 standard.

Reply:
`PyCharm` did not correct me), now I will record a video and create a new pool of request